commands/module-load.md

```diff
@@ -0,0 +1,14 @@
+Loads a module from a dynamic library at runtime.
+
+This command loads and initializes the Redis module from the dynamic library
+specified by the `path` argument. The `path` should be the absolute path,
+including the full filename, of the library. Any additional arguments are passed
```

Comment:
I'd reword this. The comma separated aside is awkward and gets in the way of understanding. One solution: If you move the aside to the end, I think it makes more sense.

Reply:
Agreed.